client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -90,6 +92,25 @@ const UserPage = createClass({
 		  </td>;
 	},
 
+	handleFilterTextChange : function(e){
+		this.setState({
+			filterString : e.target.value
+		});
+		return;
+	},
+
+	renderFilterOption : function(){
+		return <td>
+			<label>
+				Filter Title:
+				<input
+					type='text'
```

Comment:
Change to `type='search'` which provides some extra functionality (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search). On Chrome, the input field gets an `X` icon in the field which when clicked clears the field, and also typing <kbd>ESC</kbd> will also clear the field.